src/components/ColumnSelector/ColumnSelectorStoriesHelper.tsx

```diff
@@ -0,0 +1,238 @@
+          role: "cell",
+          "aria-label": MEMORY, // needed
+          style: { width: `${COLUMN_WIDTHS[MEMORY]}px` },
+          onClick: () => console.log("Click on instance", instance.memory),
+        },
+        {
+          content: instance.fileSystem,
+          className: "clickable-cell",
+          role: "cell",
+          "aria-label": FILESYSTEM, // needed
+          style: { width: `${COLUMN_WIDTHS[FILESYSTEM]}px` },
+          onClick: () => console.log("Click on instance", instance.fileSystem),
+        },
+        {
+          content: (
+            <div className="u-truncate" title={instance.description}>
+              {instance.description}
+            </div>
+          ),
+          className: "clickable-cell",
+          role: "cell",
+          "aria-label": DESCRIPTION, // needed
+          style: { width: `${COLUMN_WIDTHS[DESCRIPTION]}px` },
+          onClick: () => console.log("Click on instance", instance.description),
+        },
+        {
+          key: `ipv4-${instance.ipv4.length}`,
+          content: instance.ipv4.join(", "),
+          className: "u-align--right clickable-cell",
+          role: "cell",
+          "aria-label": IPV4, // needed
+          style: { width: `${COLUMN_WIDTHS[IPV4]}px` },
+          onClick: () => console.log("Click on instance", instance.ipv4),
+        },
+        {
+          key: `ipv6-${instance.ipv6.length}`,
+          content: instance.ipv6.join(", "),
+          className: "clickable-cell",
+          role: "cell",
+          "aria-label": IPV6, // needed
+          style: { width: `${COLUMN_WIDTHS[IPV6]}px` },
+          onClick: () => console.log("Click on instance", instance.ipv6),
+        },
+        {
+          content: instance.snapshots?.length ?? "0",
+          className: "u-align--right clickable-cell",
+          role: "cell",
+          "aria-label": SNAPSHOTS, // needed
+          style: { width: `${COLUMN_WIDTHS[SNAPSHOTS]}px` },
+          onClick: () =>
+            console.log("Click on instance", instance.snapshots?.length),
+        },
+        {
+          content: instance.status,
+          role: "cell",
+          className: "clickable-cell",
+          "aria-label": STATUS, // needed
+          style: { width: `${COLUMN_WIDTHS[STATUS]}px` },
+          onClick: () => console.log("Click on instance", instance.status),
+        },
+      ],
+    };
+  });
+
+  return instanceRows;
+};
+
+export const removeHiddenColumns = (
+  rows: MainTableRow[],
+  hiddenCols: string[],
+): MainTableRow[] => {
+  return rows.map((row) => {
+    return {
+      ...row,
+      columns: row.columns.filter(
+        (item) => !hiddenCols.includes(item["aria-label"]),
+      ),
+    };
+  });
+};
+
+export const getHeaders = (hiddenCols: string[]) =>
+  COLUMN_HEADERS.map((t) => {
+    return {
+      content: t,
+      sortKey: t,
+      style: { width: `${COLUMN_WIDTHS[t]}px` },
+    };
+  }).filter((item) => !hiddenCols.includes(item.content));
```

Comment:
I think both of these functions should be exported in the library. Because client applications will need to call them to apply the hiding. Maybe move them to a dedicated file and include them in the index.ts exports?